datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -6296,6 +6299,54 @@ physical_plan
 01)AggregateExec: mode=Single, gby=[], aggr=[count(NULL)]
 02)--DataSourceExec: partitions=1, partition_sizes=[1]
 
+statement count 0
+drop table t;
+
+# test duplicated shema name issue
+
+statement count 0
+create table t (a int) as values (1), (2);
+
+query I
+select count() from t;
+----
+2
+
+query I
+select count(1) * count(2) from t;
+----
+4
+
+query I
+select count(1) * count(*) from t;
+----
+4
+
+query I
+select count(*) * count(*) from t;
+----
+4
+
+query I
```

Comment:
Likewise here it would be nice to have `count(1)` and `count(2)` individually tested